spec/services/importers/court_case_spec.rb

```diff
@@ -13,13 +15,34 @@
       # Run importer
       described_class.new(county.id, 'CF-2018-1016').perform
       court_case = CourtCase.find(court_case.id) # Pull updated object
-
-      expect(court_case.parties.count).to eq 3
-      expect(court_case.counts.count).to eq 5
+      expect(court_case.parties.count).to eq 4
+      expect(court_case.counts.count).to eq 9
       expect(court_case.current_judge.name).to eq 'CF Docket E'
       expect(court_case.events.count).to eq 30
       expect(court_case.counsels.count).to eq 1
       expect(court_case.docket_events.count).to eq 247
     end
+
+    it 'saves party with text only for a case' do
+      # Create a case with html from fixture
+      county = create(:county, name: 'oklahoma')
+
+      create(:judge, name: 'Collins, April')
+      court_case = create(:court_case, county_id: county.id, case_number: 'CS-2022-1020')
+      html = File.read('spec/fixtures/oscn_id_less.html')
+      create(:case_html, court_case: court_case, html: html)
```

Comment:
These tests look good but the standard in rspec is to create objects in the let! blocks (I'll show you how if you want)
This is fine for now